Modules/Filtering/ImageGrid/test/itkWarpImageFilterTest2.cxx

```diff
@@ -167,20 +166,14 @@ itkWarpImageFilterTest2(int, char *[])
       return EXIT_FAILURE;
     }
   }
-  if (streamIt.IsAtEnd() != it2.IsAtEnd())
-  {
-    std::cout << "Iterators don't agree on end of image" << std::endl;
-    return EXIT_FAILURE;
-  }
+
+  ITK_TEST_EXPECT_EQUAL(streamIt.IsAtEnd(), it2.IsAtEnd());
+
 
   // this verifies that the pipeline was executed as expected along
   // with correct region propagation and output information
-  if (!monitor2->VerifyAllInputCanStream(4))
-  {
-    std::cout << "Filter failed to execute as expected!" << std::endl;
-    std::cout << monitor2;
-    return EXIT_FAILURE;
-  }
+  ITK_TEST_EXPECT_TRUE(!monitor2->VerifyAllInputCanStream(4));
```

Comment:
`!` seems to be wrong here

Reply:
Had spotted that. Thanks for the heads-up. Fixed in a36b621.